test/peripheral/521_borrowProxy_base.ts

```diff
@@ -169,21 +171,21 @@ contract('BorrowProxy', async (accounts) => {
           WETH,
           maturity1,
           user2,
+          oneToken.mul(2),
```

Comment:
Erring on the side of not changing regression tests, let's leave tests like this have the same values

```suggestion
          oneToken,
```

Reply:
Fixed